src/components/experiment-tracking/runs-list-card/runs-list-card.js

```diff
@@ -15,27 +19,47 @@ const RunsListCard = ({
   disableRunSelection = false,
   enableComparisonView = false,
   onRunSelection,
-  selectedRuns = [],
+  selectedRunIds = [],
 }) => {
   const { id, timestamp, title = null, bookmark } = data;
   const [active, setActive] = useState(false);
+  const [updateRunDetails] = useMutation(UPDATE_RUN_DETAILS, { client });
   const humanReadableTime = toHumanReadableTime(timestamp);
 
-  const onClick = (id) => {
+  const onClick = (id, e) => {
```

Comment:
please rename this to something more descrption such as `onToggleBookmarkButton`, etc

Reply:
This function can either toggle the bookmark or fire the `onRunSelection` function, so it isn't strictly about the bookmark. 

I can change it to `onRunsListCardClick`. Is that ok?